src/main/resources/api-specs/review-openapi.yaml

```diff
@@ -0,0 +1,185 @@
+          in: "query"
+          required: false
+          schema:
+            type: "integer"
+            format: "int32"
+            default: 0
+            minimum: 0
+        - name: "size"
+          description: "The number of reviews per page."
+          in: "query"
+          required: false
+          schema:
+            type: "integer"
+            format: "int32"
+            default: 50
+            minimum: 1
+            maximum: 100
+        - name: "sort_attribute"
+          description: "The name of the product field by which the results should be sorted."
+          in: "query"
+          required: false
+          schema:
+            type: "string"
+            default: "createdAt"
+        - name: "sort_direction"
+          description: "The direction of the sorting."
+          in: "query"
+          required: false
+          schema:
+            type: "string"
+            default: "desc"
+      responses:
+        "200":
+          description: "The product was found and its reviews are returned."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReviewsAndRatingsWithPagination"
+    post:
+      tags:
+        - "Review"
+      summary: "Add a review item for a product by authorized user"
+      operationId: "addReview"
+      parameters:
+        - name: "productId"
+          in: "path"
+          required: true
+          description: "Unique identifier of the product."
+          schema:
+            type: "string"
+            format: "uuid"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NewReview"
+        required: true
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReviewResponse"
+
+  /api/v1/products/{productId}/reviews/{reviewId}:
+    delete:
+      tags:
+        - "Review"
+      summary: "Remove a review item for a product by authorized user"
+      operationId: "deleteReview"
+      parameters:
+        - name: "productId"
+          in: "path"
+          required: true
+          description: "Unique identifier of the product."
+          schema:
+            type: "string"
+            format: "uuid"
+        - name: "reviewId"
+          in: "path"
+          required: true
+          description: "Unique identifier of the review."
+          schema:
+            type: "string"
+            format: "uuid"
+      responses:
+        "200":
+          description: "OK"
+
+components:
+  schemas:
+    NewReview:
+      type: "object"
+      properties:
+        text:
+          type: "string"
+          minLength: 1
+          maxLength: 1500
+      required:
+        - text
+
+    ReviewResponse:
+      type: "object"
+      properties:
+        reviewId:
+          type: "string"
```

Comment:
You can write here "format: "uuid" and don't convert id to string in converter

Reply:
Fixed